pkg/builder/builder.go

```diff
@@ -230,8 +231,17 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	steps = append(steps,
 		&stepMountImage{PartitionsKey: "partitions", ResultKey: "mount_path"},
 		&StepMountExtra{ChrootKey: "mount_path"},
-		&stepQemuUserStatic{ChrootKey: "mount_path", PathToQemuInChrootKey: "qemuInChroot", Args: Args{Args: b.config.QemuArgs}},
-		&stepRegisterBinFmt{QemuPathKey: "qemuInChroot"},
+	)
+
+	native := runtime.GOARCH == "arm" || runtime.GOARCH == "arm64"
```

Comment:
Surely native needs to be true if, and only if `GOARCH` is the same as target? E.g. if the target is arm64, but the host is arm (and possibly vice versa), you would still need binfmt.